commands/activation.go

```diff
@@ -25,7 +25,7 @@ import (
     "time"
 
     "github.com/openwhisk/openwhisk-client-go/whisk"
-    "github.com/openwhisk/openwhisk-cli/wski18n"
+    "../wski18n"
```

Comment:
Should we stick to gradle distDocker or use go build? 

Go build fails for me at commands/property.go with commands/property.go:24:5: cannot find package "github.com/mitchellh/go-homedir"